src/website/views/components/avatar.pug

```diff
@@ -0,0 +1,323 @@
+---
+title: Avatar
+---
+
+- var sizes = ['xs', 'sm', 'sm2', 'sm3', 'md', 'lg', 'xl', 'xxl'];
+- var columnWidths = {'xs':'1', 'sm':'1', 'sm2':'1', 'sm3':'1', 'md':'1', 'lg':'2', 'xl':'2', 'xxl':'3'};
+- var colors = ["grey", "red", "pink", "purple", "indigo", "navy", "blue", "teal", "emerald", "green", "yellow", "orange"];
+
+- var iconExamples = ['user', 'users', 'star', 'visible', 'rocket'];
+
+p.-text An avatar is a graphical representation of the user. It can be build from a photo, an icon or one or two letters.
+
+h3 Using avatars
+
+p.-text
+  | To render an avatar, use the class <code>a-avatar</code> in a div element and put an image inside. The image
+  | must be square in order not to be stretched.
+.example.-mb3
+  .-p3
+    div(class=`a-avatar`)
+      img(src='../../assets/images/avatar.jpg')(alt="avatar")
```

Comment:
Let's use a regular photo without an effect applied. Initially thought it was a CSS bug. You're welcome to use my GitHub photo as there's no face.